medusa/server/web/config/providers.py

```diff
@@ -108,7 +108,7 @@ def getNewznabCategories(name, url, api_key):
         # Get newznabprovider obj with provided name
         temp_provider = NewznabProvider(name, url, api_key)
 
-        success, tv_categories, caps, error = temp_provider.get_categories()
+        success, tv_categories, caps, error = temp_provider.get_capabilities()
 
         return json.dumps({'success': success, 'tv_categories': tv_categories, 'caps': caps, 'error': error})
```

Comment:
Then you could easily change to json by doing: json.dumps(named_tuple._asdict())